DevTools/Playground/main.cpp

```diff
@@ -300,6 +301,35 @@ int main(int argc, char** argv)
         app->quit();
     }));
 
+    auto& edit_menu = menubar->add_menu("Edit");
+    edit_menu.add_action(GUI::Action::create("Format GML", { Mod_Ctrl | Mod_Shift, Key_I }, [&](auto&) {
+        auto source = editor.text();
+        GUI::GMLLexer lexer(source);
+        for (auto& token : lexer.lex()) {
+            if (token.m_type == GUI::GMLToken::Type::Comment) {
+                auto result = GUI::MessageBox::show(
+                    window,
+                    "Your GML contains comments, which currently are not supported by the formatter and will be removed. Proceed?",
+                    "Warning",
+                    GUI::MessageBox::Type::Warning,
+                    GUI::MessageBox::InputType::OKCancel);
+                if (result == GUI::MessageBox::ExecCancel)
+                    return;
+                break;
+            }
+        }
+        auto formatted_gml = GUI::format_gml(source);
+        if (!formatted_gml.is_null()) {
+            editor.set_text(formatted_gml);
```

Comment:
Future change idea: Maybe the editor could have a formatter (just like a highlighter and completion provider), and we could later enable "format on save" or "format on change"?

Obviously not a requirements for this PR :P